user-guide/Reference/faq/DataMiner_client_applications.md

```diff
@@ -58,6 +58,9 @@ To reinstall the browser app, you need to clean the XBAP cache:
 
 ## What should I do if I cannot connect to a DMA using the DataMiner Cube browser app?
 
+> [!NOTE]
+> DataMiner Cube can either be used as a desktop application (recommended) or as a browser application in Internet Explorer or Edge. See [Installing the DataMiner Cube desktop application](xref:Installing_the_DataMiner_Cube_desktop_application)
```

Comment:
Please rephrase the note to state more clearly that the desktop application is recommended and to not mention Internet Explorer (as IE is no longer available in recent Windows versions):
> While using the DataMiner Cube browser application is still supported, we highly recommend using the desktop application instead. See [Installing the DataMiner Cube desktop application](xref:Installing_the_DataMiner_Cube_desktop_application).